framework/src/parser/Builder.C

```diff
@@ -1210,9 +1212,6 @@ Builder::extractParams(const std::string & prefix, InputParameters & p)
         setvectorvector(BoundaryID);
         setvectorvector(Point);
         setvectorvector(string);
-        setvectorvector(FileName);
-        setvectorvector(FileNameNoExtension);
-        setvectorvector(MeshFileName);
```

Comment:
the issue with removing those is that the person trying to add a vector of vector is not going to understand why it fails until they reach out to one of like 4 people who know about this

Reply:
The thing is it's not supported now anyway; the proper backend stuff isn't setup for those parameters. And you do get an error that says it's not supported

Reply:
We could likely make this a compile time error when someone tries to add a parameter that isn't supported

Reply:
sounds good